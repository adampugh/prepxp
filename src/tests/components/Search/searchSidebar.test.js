import React from "react";
import { shallow } from "enzyme";

import SearchContent from "../../../components/Search/searchContent";
import lists from "../../fixtures/searchLists";

describe("<searchContent />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<SearchContent 
            searchLists={lists.filteredLists} 
            filterByTag={() => {}}
            clearFilter={() => {}} />);
        expect(wrapper).toMatchSnapshot();
    });
});