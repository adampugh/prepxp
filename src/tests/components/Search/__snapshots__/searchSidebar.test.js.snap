// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<searchContent /> should match snapshot 1`] = `
<div
  className="searchContent"
>
  <div
    className="container-l"
  >
    <div
      className="searchContent__grid"
    >
      <SearchSidebar
        clearFilter={[Function]}
        filterByTag={[Function]}
        searchLists={
          Array [
            Object {
              "list": Object {
                "questions": Array [
                  1,
                  2,
                  3,
                  4,
                ],
                "title": "Angular Developer",
              },
              "title": "Angular Developer",
            },
            Object {
              "list": Object {
                "questions": Array [
                  1,
                  2,
                  5,
                ],
                "title": "Node JS Developer",
              },
              "title": "Node JS Developer",
            },
            Object {
              "list": Object {
                "questions": Array [
                  1,
                  1,
                  1,
                  1,
                  1,
                  1,
                  1,
                ],
                "title": "General Interview Questions 1",
              },
              "title": "General Interview Questions 1",
            },
          ]
        }
      />
      <searchGrid />
    </div>
  </div>
</div>
`;
